### DISCUSSION TITLE
when i build source code, i have some difficult, need help!!!

### DISCUSSION THREAD

**Discussion by qigeqigeqi** [coder/code-server#4007] [Q&A]
when i run           node .            

there is problem：
internal/modules/cjs/loader.js:1131
  return process.dlopen(module, path.toNamespacedPath(filename)
Error: /lib64/libc.so.6: version `GLIBC_2.25' not found (required by /home/opt/code-server-3.11.1/release/node_modules/argon2/lib/binding/napi-v3/argon2.node)
at Object.<anonymous> (/home/opt/code-server-3.11.1/release/node_modules/argon2/argon2.js:9:56)

how i can update GLIBC, now there is GLIBC_2.17 in my system

**Comment by jsjoeio** [comment]
Hmm... It sounds like it's looking for `GLIBC_2.25` which is required by `argon2` which is what we use for password hashing. 

What OS are you using? 

If you're using CentOS, here's an article about updating glibc: https://www.liquidweb.com/kb/how-to-update-the-glibc-gnu-libc-in-centos-red-hat/
